docs/data_usage_guides/endpoint_parameters/resource_parameters.yaml

```diff
@@ -0,0 +1,102 @@
+- Parameter: '`hdx_id`'
```

Comment:
Similar to the dataset endpoint, some field names have changes here as well: https://github.com/OCHA-DAP/hdx-hapi/pull/133

hdx_id -> resource_hdx_id
dataset_title -> dataset_hdx_title